functional/agent_UUID_assignment_options/test.sh

```diff
@@ -49,15 +49,18 @@ rlJournalStart
         rlRun "limeStopAgent"
     rlPhaseEnd
 
+    # disable agent_UUID_assignment_options test regarding to bz#2165055
+    if [ -n "$UUID_PHASE" ]; then
```

Comment:
We are not working with this variable, better use simply `if false; then` to make it obvious the phase is intentionally disabled.

Reply:
Yeah, thanks that's more handy. I added variable due to manual testing, if we setting variable in plan we could run this subtest. But nevermind, we can simply remove the conditional.

Reply:
Right, I got its purpose. But the thing is, in order to understand what `UUID_PHASE` variable defined in a test plan really means, one would have to remember it relates to this specific test end enables/disables a subtest. IMHO we would have give this variable more descriptive name and even that, I am not sure it is really worth it.